cellcounter/cc_kapi/factories.py

```diff
@@ -1,13 +1,13 @@
 import string
-
-import factory
-from django.contrib.auth.models import User
+from django.contrib.auth.models import User, AnonymousUser
 
 from cellcounter.main.models import CellType
-from . import models
+from .models import Keyboard, KeyMap, DefaultKeyboards
```

Comment:
You've changed the import, but not all the references below it (models.KeyMap is still in play)